app/models/searchs/vacancy.rb

```diff
@@ -3,45 +3,15 @@ def initialize(repository = ::Vacancy)
     @repository = repository
   end
 
-  def list(query = nil, lang = nil)
+  def list(query = nil)
     return @repository.recent if query.blank?
 
-    lang ||= I18n.locale == :'pt-BR' ? "portuguese" : "english"
+    query = query.split(' ').map { |entry| "#{entry}:*" }.join(' & ')
```

Comment:
The default parameter for `.split` is whitespace :)

_"If pattern is omitted, the value of $; is used. If $; is nil (which is the default), str is split on whitespace as if ` ‘ were specified."_
http://apidock.com/ruby/String/split